iosHyperskillApp/iosHyperskillApp/Sources/Views/SwiftUI/PlaceholderView/PlaceholderView.swift

```diff
@@ -16,15 +16,15 @@ struct PlaceholderView: View {
                 }
 
                 if let title = configuration.title,
-                   let text = title.text.trimmed(),
+                   let text = Optional(title.text.trimmed()),
```

Comment:
🤔

Reply:
У меня обновился xcode и появилась ошибка "Initializer for conditional binding must have Optional type"

Reply:
While developing locally we must use the same version of the Xcode (14.2) as in our GitHub Actions. This will guarantee that locally and in GitHub Actions, we use the same toolchain (Swift version, etc...)